.ci/Jenkinsfile.gosh-prepare-rc

```diff
@@ -223,22 +224,18 @@ pipeline {
                     dir('monitoring/gosh-mon') {   
                         sh "cp config/credentials.yml.template config/credentials.yml"
                         sh "docker build -t gosh-mon ."
-                        def tests = sh(script: 'cat config/scripts/app.yml | cut -c 3- | tr -d \'^M\' | sed -n \'/:/!p\'', returnStdout: true).trim().split('\n')
-                        for (int i = 0; i < tests.length; i++) {
-                            stage("Test ${tests[i]}") {
-                                sh """
-                                    docker run --rm \
-                                        -e GM_MODE='${tests[i].trim()}' \
-                                        -e RUN_NOW=1 \
-                                        -e CONFIG_SEED="\$(tonos-cli -j genphrase | jq .phrase | tr -d '\"\n')" \
-                                        -e CONFIG_APPURL=https://${BUCKET_NAME}.gosh.sh/ \
-                                        -e CONFIG_REDIS_HOST="" \
-                                        -v \$(pwd)/config:/home/node/app/config:ro \
-                                        -v \$(pwd)/test-errors:/home/node/app/errors \
-                                        gosh-mon
-                                """
-                            }
-                        }
+                        sh """
+                            docker run --rm \
+                                -e GM_MODE='app-setup' \
```

Comment:
The problematic test `root-check` was removed from `app.yml`, maybe better return all tests in order?